gf-guide/metadata.md

```diff
@@ -77,6 +76,7 @@ source {
   commit: "d9098c0a72125d411dbb225a2e5a61dc15265ffc"
 }
 minisite_url: "https://myfontwebsite.com"
+stroke: "SANS_SERIF"
```

Comment:
AFAIK if the `SANS_Serif` is already defined as the category, there is no need for a `stroke` field for it

Reply:
I'm looking at the emails from when this was set up, and it seems like stroke should be set if it accurately describes the design. I don't see a problem with including it the way it is here.

It's better to include it, some features in the UI only looks at the stroke value for some things.

The stroke field has a single value that is one of Sans Serif, Serif, Slab Serif, or it can not be set. It should always be set if one of those options accurately describes the design.

Reply:
Yes, thanks! Doing that now.